compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/mouse/MouseScrollFilter.skiko.kt

```diff
@@ -114,7 +114,7 @@ fun Modifier.mouseScrollFilter(
     //  the more proper behaviour would be to batch multiple scroll events into the single one
     while (true) {
         val event = awaitScrollEvent()
-        val mouseEvent = (event.mouseEvent as? MouseWheelEvent) ?: continue
```

Comment:
We don't send a native event via ImageComposeScene, so we should provide default values for scrolling settings